Resources/Prototypes/Entities/Objects/Weapons/Guns/Basic/base_wieldable.yml

```
@@ -10,4 +10,6 @@
     maxAngle: -30
   - type: Gun
     minAngle: 21
-    maxAngle: 32   
+    maxAngle: 32
+    # WL-Changes: CombatStandSystem (из двуручного оружия нельзя стрелять лёжа)
+  - type: GunStandingRequired
```

```suggestion    
  - type: GunStandingRequired # WL-Changes: CombatStandSystem (из двуручного оружия нельзя стрелять лёжа) 
```
Что-то тут разметка уехала...